<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestionar_VEstudiante</class>
 <widget class="QMainWindow" name="Gestionar_Estudiante">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de Estudiantes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="estudiantes_tbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Cédula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Editar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="agregar_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelar_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscar_txt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buscar_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gestión de Estudiantes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEstudiante">
    <property name="title">
     <string>Estudiante</string>
    </property>
    <addaction name="agregar_menu"/>
    <addaction name="separator"/>
    <addaction name="salir_menu"/>
   </widget>
   <addaction name="menuEstudiante"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="agregar_menu">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="salir_menu">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
